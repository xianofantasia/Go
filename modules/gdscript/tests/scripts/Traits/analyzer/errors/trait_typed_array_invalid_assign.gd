trait SomeTrait:
	pass

class OtherClass:
	pass

func test():
	var trait_classes: Array[SomeTrait] = [OtherClass.new()]
