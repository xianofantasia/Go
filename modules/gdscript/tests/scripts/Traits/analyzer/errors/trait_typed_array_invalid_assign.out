GDTEST_ANALYZER_ERROR
>> ERROR at line 8: Cannot have an element of type "OtherClass" in an array of type "Array[SomeTrait]".
